l10n_it_fatturapa_pec/models/mail_thread.py

```diff
@@ -168,60 +165,34 @@ def find_attachment_by_subject(self, subject):
         return attachment_out_model.browse()
 
     def create_fatturapa_attachment_in(self, attachment, message_dict=None):
-        decoded = base64.b64decode(attachment.datas)
-        fatturapa_attachment_in = self.env['fatturapa.attachment.in']
         fetchmail_server_id = self.env.context.get('fetchmail_server_id')
         received_date = False
         if message_dict is not None and 'date' in message_dict:
             received_date = message_dict['date']
         company_id = False
-        e_invoice_user_id = False
         # The incoming supplier e-bill doesn't carry which company
         # we must use to create the given fatturapa.attachment.in record,
         # so we expect fetchmail_server_id coming in the context
         # see fetchmail.py.
         # With this information we search which SDI is actually using it,
         # finally the SDI contain both company and user we would need to use
+        sdi_channel_model = self.env['sdi.channel']
         if fetchmail_server_id:
-            sdi_chan = self.env['sdi.channel'].search([
+            sdi_chan = sdi_channel_model.search([
```

Comment:
In `create_fatturapa_attachment_in` `e_invoice_user_id` non viene più considerato?

Nei DB multi-company questa ricerca potrebbe comportare errori di sicurezza

Reply:
> Viene già considerato durante la creazione di una nuova FE

Cioè come?

Reply:
Grazie